azure_arc_servers_jumpstart/gcp/terraform/main.tf

```diff
@@ -0,0 +1,89 @@
+
+// Terraform plugin for creating random ids
+resource "random_id" "instance_id" {
+ byte_length = 8
+}
+
+// An Azure Resource Group
+resource "azurerm_resource_group" "azure_rg" {
+  name     = var.azure_resource_group
+  location = var.azure_location
+}
+
+// A single Google Cloud Engine instance
+resource "google_compute_instance" "default" {
+ name         = "gcp-vm-${random_id.instance_id.hex}"
```

Comment:
We need the name to be more explicit and demo-oriented. Something like "Arc-GCP-Demo"

Reply:
fixed, note that GCP naming doesnt allow uppercase